order-api/src/main/java/com/smoothstack/order/Main.java

```diff
@@ -51,5 +53,4 @@ public int getExitCode() {
         }
 
     }
-
 }
```

Comment:
CORS mappings belong in the Shared security module. It's already there actually.